app/controllers/api/authentications_controller.rb

```diff
@@ -1,4 +1,18 @@
 module Api
   class AuthenticationsController < BaseController
+    def delete_resource(type, id, _data = {})
+      auth = resource_search(id, type, collection_class(:authentications))
+      raise 'type not currently supported' unless auth.respond_to?(:delete_in_provider_queue)
```

Comment:
Maybe update the message. maybe something like "Delete not supported for #{authentication_ident(auth)}"